packages/perseus/src/perseus-types.ts

```diff
@@ -762,7 +762,7 @@ export type PerseusNumericInputAnswer = {
     // The forms available for this answer.  Options: "integer, ""decimal", "proper", "improper", "mixed", or "pi"
     // NOTE: perseus_data.go says this is required even though it isn't necessary.
     answerForms?: ReadonlyArray<MathFormat>;
-    // Whether the answerForms should be strictly matched
+    // Whether validatorForms should include the defaultAnswerForms, or strictly match to the provided answerForms
```

Comment:
I'd like to keep the original terms here.  The current changes focus on the code, but these types are more like the schema for Perseus.  So it'd be great if they could be more "end user" focused to describe how these settings affect the question.  

Maybe something like this?

>  Whether we should check the answer strictly against the the configured answerForms (strict = true) or include a set of default answerForms (strict = false).